pkg/assembler/backends/inmem/hasSBOM.go

```diff
@@ -192,6 +258,29 @@ func (c *demoClient) convHasSBOM(in *hasSBOMStruct) (*model.HasSbom, error) {
 		}
 		out.Subject = c.convArtifact(art)
 	}
+	// TODO(knrc) - do we want to return full objects for the following?
```

Comment:
Based on how `model.HasSBOM` is defined:

```
type HasSBOM {
  id: ID!
  "SBOM subject"
  subject: PackageOrArtifact!
  "Identifier for the SBOM document"
  uri: String!
  "Algorithm by which SBOMs digest was computed"
  algorithm: String!
  "Digest of SBOM"
  digest: String!
  "Location from which the SBOM can be downloaded"
  downloadLocation: String!
  "Document from which this attestation is generated from"
  origin: String!
  "GUAC collector for the document"
  collector: String!
  "Timestamp for SBOM creation"
  knownSince: Time!
  "Included packages and artifacts"
  includedSoftware: [PackageOrArtifact!]!
  "Included dependencies"
  includedDependencies: [IsDependency!]!
  "Included occurrences"
  includedOccurrences: [IsOccurrence!]!
}
```
we will have to return the objects

Reply:
This has been updated and pushed